app/models/server/models/Uploads.js

```diff
@@ -21,9 +21,7 @@ export class Uploads extends Base {
 			doc.instanceId = InstanceStatus.id();
 		});
 
-		this.tryEnsureIndex({ rid: 1 });
-		this.tryEnsureIndex({ uploadedAt: 1 });
-		this.tryEnsureIndex({ typeGroup: 1 });
+		this.tryEnsureIndex({ rid: 1, uploadedAt: -1, typeGroup: 1, _hidden: 1 });
```

Comment:
I'm not sure this is the optimal index yet 🤔 

anyways, if we decide to remove the old indexes, we should include a `dropIndex` in the migration.